assembly/__tests__/Int.spec.ts

```diff
@@ -59,19 +59,13 @@ describe("Int8", () => {
         expect<Int8>(Int8.fromU8a([0x81])).toStrictEqual(new Int8(-127));
     });
 
-    itThrows("should throw when decoding i16", () => {
-        let v1 = Int8.fromU8a([0x01, 0]);
-        let v2 = Int8.fromU8a([0x01, 0xc0]);
+    it("should decode only first byte", () => {
+        expect<Int8>(Int8.fromU8a([0xff, 0])).toStrictEqual(new Int8(-1));
+        expect<Int8>(Int8.fromU8a([0, 0, 1, 0xf1, 12], 3)).toStrictEqual(new Int8(-15));
```

Comment:
We could have a failing one as-well in another `it`.
[1, 2, 3, 4]; index: 1 for INT8 should work, but 
[1, 2, 3, 4]; index: 4 for INT should throw.

similarly, the other types could have such tests as-well